apps/server/src/mev/model/Mev.ts

```diff
@@ -57,3 +59,52 @@ export interface Mev {
   solAmountDrained: number;
   transactionHash: string;
 }
+
+export enum MevAttackSwapType {
+  BUY = "BUY",
+  SELL = "SELL",
+}
+
+export interface MevTransaction {
+  address: string;
+  signature: string;
+}
+
+export class MevAttackResponse {
```

Comment:
imo because this is single DTO, Response could be wrapper of all items, but I think not needed now 🙏 

```suggestion
export class MevAttack {
```